src/utility.ts

```diff
@@ -139,15 +141,24 @@ export const Utility = {
 			});
 		}
 	},
-
-	objToJsonString(obj: unknown): string {
+	// eslint-disable-next-line @typescript-eslint/ban-types
+	objToJsonString(obj: object): string {
 		const JSONbigNative = json_bigint({ useNativeBigInt: true });
 		return JSONbigNative.stringify(obj);
 	},
 
-	jsonStringToObj<T>(json: string): T {
+	jsonStringToObj<T>(json: string, config: TigrisClientConfig): T {
```

Comment:
Could you also add some comments here and the above method? Also, for this method talk about the usage of the config.

Reply:
yes sounds good. added comment.